packages/app/src/hooks/useRowWhere.tsx

```diff
@@ -66,6 +67,52 @@ export default function useRowWhere({
                 value,
                 chType,
               ]);
+            case JSDataType.JSON:
+              // Handle case for whole json object, ex: json
+              return SqlString.format(`lower(hex(MD5(toString(?))))=?`, [
+                SqlString.raw(valueExpr),
+                MD5(value).toString(),
+              ]);
+            case JSDataType.Dynamic:
+              // Handle case for json element, ex: json.c
+
+              // Currently we can't distinguish null or 'null'
+              if (value === 'null') {
+                return SqlString.format(`isNull(??)`, SqlString.raw(column));
+              }
+              if (
+                value.length > 1000 ||
+                column.length > 1000 ||
+                columJsonSplit.length >= 8
+              ) {
+                throw new Error(
+                  'Search value/object key too large, or nested too many level',
+                );
+              }
```

Comment:
curious about the reason to set up this cap

Reply:
It is like the limitation at [here](https://github.com/hyperdxio/hyperdx/blob/bef11df3c36b380173644418034b3054a057d999/packages/app/src/hooks/useRowWhere.tsx#L71).

In string we can `md5` if its too long, but for `json`, because we don't know dynamic type, md5 will give wrong result.

`column` is to prevent customer have huge key, `columJsonSplit` is to prevent json become too deep, affect performance, I will remove this since we don't do `simpleJSONExtractRaw ` for `array` and `object` now.